src/_includes/scripts/index.js

```diff
@@ -0,0 +1,134 @@
+function init() {
+  var analyticsCookieName = "modernising_cookie_policy";
+  var gaCookieValue = getCookie(analyticsCookieName);
+  var gaCookieSettings = document.querySelector(
+    'input[name="ga-preferences"]:checked'
+  );
+  var isAnalyticsCookieSet = gaCookieValue !== null;
+  document.addEventListener("click", (e) => {
+    if (e.target.name === "cookies") {
+      if (e.target.value === "update") {
+        var newCookieSetting = document.querySelector(
+          'input[name="ga-preferences"]:checked'
+        );
+        if (newCookieSetting !== null && newCookieSetting.value !== null) {
+          setCookie(analyticsCookieName, newCookieSetting.value === "true");
+        }
+        document.location.href = "/";
+      }
+      if (e.target.value === "accept") {
+        setCookie(analyticsCookieName, true);
+        toggleCookieBanner(true);
+        toggleCookieBannerConfirm(false);
+      }
+      if (e.target.value === "reject") {
+        setCookie(analyticsCookieName, false);
+        toggleCookieBanner(true);
+        toggleCookieBannerReject(false);
+      }
+      if (e.target.value === "hide") {
+        toggleCookieBannerConfirm(true);
+        toggleCookieBannerReject(true);
+      }
+    }
+
+    if (isAnalyticsCookieSet) {
+      if (e.target) {
```

Comment:
What's the browser target? Might be able to simplify this with optional chaining.

Reply:
Officially IE11 I believe but decided on making it all as simple as possible to increase browser coverage. Same reason I didn't go down a module route